src/components/Card/__snapshots__/Card.test.jsx.snap

```diff
@@ -3,7 +3,7 @@
 exports[`should render the text on the display 1`] = `
 <div>
   <div
-    class="card"
+    class="card card--undefined"
```

Comment:
update code so these don't come through as undefined.. either update the tests to have a prop filled, or update the component so there's a default value